### ISSUE TITLE
Operator Registration email was not sent

### ISSUE THREAD

**Issue by TroyManary** [TroyManary/EasyPlow#77]
Paty created an operator account on easyplow.com but did not receive the registration email.  It wasn't in her Junk folder either.

This also happened to me several weeks ago when I first started testing.  Since then, it has worked OK every time I registered a new account (which I have done dozens of times).

Do we have a bug here?
We will keep testing to see if we can figure out what triggers it.

Setting to Low priority for now.

**Comment by paty-eplow** [comment]
Use another real address (not dummy address for testing purposes), and the confirmation email was immediately. then proceed to continue with cooperator registration.

**Comment by gustave-eplow** [comment]
Yeah - this has happened to me a few times as well. Not recently; maybe a
month ago...

**Comment by AlexEdimensionz** [comment]
The email may not be in the Junk folder of her email app. It may have been filtered out at the email service level and end up in that spam folder through web log in. Tested, there is no condition under which email doesn't get sent.

**Comment by TroyManary** [comment]
Put State to monitor so we can keep a record of this, but not actively working on it.

**Comment by TroyManary** [comment]
We found Paty's lost e-mail from 25-Sep.  It was rejected by Outlook.com.
This email was sent to contact@easyplow.com:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a permanent error. The following address(es) failed:

  paavenda@dal.ca
    host dal-ca.mail.protection.Outlook.Com [207.46.163.170]
    SMTP error from remote mail server after RCPT TO:<paavenda@dal.ca>:
    550 5.7.1 Service unavailable; Client host [69.16.237.19] blocked using FBLW15; To request removal from this list please forward this message to delist@messaging.microsoft.com

------ This is a copy of the message, including all the headers. ------

Return-path: <contact@easyplow.com>
Received: from eplow by host.e-plow.com with local (Exim 4.85)
        (envelope-from <contact@easyplow.com>)
        id 1ZfCTH-0004il-TY
        for paavenda@dal.ca; Thu, 24 Sep 2015 15:46:23 -0400
To: paavenda@dal.ca
Subject: Account Details for Lophius Plow at EasyPlow
Date: Thu, 24 Sep 2015 19:46:23 +0000
From: EasyPlow <contact@easyplow.com>
Message-ID: <d0ac3409daa67157db83deaa5769fd01@www.easyplow.com>
X-Priority: 3
X-Mailer: PHPMailer 5.2.9 (https://github.com/PHPMailer/PHPMailer/)
MIME-Version: 1.0
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: 8bit

Hello Pat,

Thank you for registering at EasyPlow. Your account is created and must be activated before you can use it.
To activate the account click on the following link or copy-paste it in your browser:
http://www.easyplow.com/en/component/users/?task=registration.activate&token=1a1cdd807a498d19d30de7af09fe3543

After activation you may login to http://easyplow.com/ using the following username and password:

Username: paavenda@dal.ca
Password: Fish123

**Comment by TroyManary** [comment]
Somehow Outlook.com (microsoft) is able to reject our e-mails to certain addresses.  We don't know why they are rejecting it, but Gustave followed the instructions and replied to delist@messaging.microsoft.com.  No we are waiting to hear back from them.

**Comment by AlexEdimensionz** [comment]
This is weird,

I checked public blacklists for the IP (69.16.237.19) and it's on none 
of them.
Whoever controls contact@easyplow.com should send an email to

delist@messaging.microsoft.com <mailto:delist@messaging.microsoft.com> 
with the following content.

"
I am the owner of easyplow..com with the IP address: 69.16.237.19

Our IP has been blocked by outlook.com and we are getting FBLW15 notices.
I have checked all public blacklists and my domain or IP do not appear 
on any of them.

How can we delist our domain and IP from your blacklist?
"




e-dimensionz Inc.
Tel: 226.444.8466
Mob: 226.338.8980
http://e-dimensionz.com

**Comment by TroyManary** [comment]
We received this response from microsoft.
Hopefully this resolves the issue.
From: Microsoft Customer Support <OLSRV.FOPE.WW.00.EN.WIP.BOM.TS.T01.DLS.ST.EM@css.one.microsoft.com>
Date: Tue, Oct 27, 2015 at 4:27 AM
Subject: RE: SRX1308670866ID - Fwd: Re: Fwd: Mail delivery failed: returning message to sender
To: easyplow.com, contact@easyplow.com


Hello ,

Thank you for contacting Microsoft Online Services Technical Support.  This email is in reference to ticket number 1308670866, which was opened in regards to your delisting request for 69.16.237.19

The IP address you submitted has been reviewed and removed from our block lists.  Please note that there may be a 1-2 hour delay before this change propagates through our entire system.

We apologize for any inconvenience this may have caused you.  As long as our spam filtering systems do not mark a majority of email from the IP address as spam-like, your messages will be allowed to flow as normal through our network.  However, should we detect an increase in spam-like activity, the IP address may be re-added to our block list.

Should you have any further questions or concerns, please feel free to respond to this email.

Thank you again for contacting Microsoft Online Services technical support and giving us the opportunity to serve you.
